docs/UserGuide.adoc

```diff
@@ -81,53 +81,68 @@ _[.small]#Fig 3.2: GUI of NOVA after user typed contact `add n\John Doe p\123456
 ==== Add Contact: `add`
 You can add your classmate or teammate as contact.
 
-Format: `add n\[name] p\[phone number] c\[classmate/teammate]`
+Format: `add n\[name] p\[phone number] e\[email address] c\[classmate/teammate]`
 
 [NOTE]
 ====
 * `[name]` is not case-sensitive. (E.g. "Jane doe" is the same as "jane Doe")
-* `[classmate/teammate]` is not case-sensitive. (E.g. "ClassMate" is the same as "classmate")
 * The name you add will be automatically formatted. (E.g. "jane doe" will become "Jane Doe")
+* `[classmate/teammate]` is not case-sensitive. (E.g. "ClassMate" is the same as "classmate")
+* You can only add either `classmate` or `teammate` as category
 ====
 
 Example: +
-`add n\John Doe p\12345678 c\classmate`
+`add n\John Doe p\12345678 e\johndoe@gmail.com c\classmate`
 
-Adds a new contact with name John Doe, phone number 12345678 into the classmate category.
+Adds a new contact with name John Doe, phone number 12345678 and email address johndoe@gmail.com into the
+classmate category.
 
 ==== Edit Contact: `edit`
 You can edit the contacts that you have added. If the contact you want to edit does not exist, NOVA will let you know.
 
-Format: `edit n\[name] p\[phone number] c\[classmate/teammate]`
+Format: `edit INDEX n\[name] p\[phone number] e\[email address] c\[classmate/teammate]`
 
 [NOTE]
 ====
+* Edits the person at the specified `INDEX`. The index refers to the index number shown in the displayed person list. +
+The index **must be a positive integer** 1, 2, 3, …
+* At least one of the optional fields must be provided.
+* Existing values will be updated to the input values.
 * `[name]` is not case-sensitive. (E.g. "Jane doe" is the same as "jane Doe")
+* The name you add will be automatically formatted. (E.g. "jane doe" will become "Jane Doe")
 * `[classmate/teammate]` is not case-sensitive. (E.g. "ClassMate" is the same as "classmate")
+* You can only edit either `classmate` or `teammate` as category
 ====
 
-Example: +
-`edit n\John Doe p\88888888 c\classmate`
+Examples:
 
-Edits the phone number of John Doe to 88888888.
+* `edit 1 p\88888888` +
+Edits the phone number of contact at index 1 to 88888888.
+
+* `edit 3 e\kly838@hotmail.com` +
+Edits the email address of contact at index 3 to kly838@hotmail.com.
 
 ==== Delete Contact : `delete`
 You can delete a contact that you have added. If the contact you try to delete does not exist, NOVA will let you know.
 
-Format: `delete n\[name]`
+Format: `delete INDEX`
 
 [NOTE]
-`[name]` is not case-sensitive. (E.g. "Jane doe" is the same as "jane Doe")
+====
+* Deletes the person at the specified `INDEX`.
+* The index refers to the index number shown in the displayed person list.
```

Comment:
Consistency milady. You want to put this line as a new bullet or put it at the back of the first point? Same for the other commands below.